src/set_theory/surreal/basic.lean

```diff
@@ -199,6 +199,16 @@ begin
   { exact numeric_one }
 end
 
+/-- Ordinal games are numeric. -/
+theorem numeric_to_pgame (o : ordinal) : o.to_pgame.numeric :=
+begin
+  induction o using ordinal.induction with o IH,
+  rw numeric_def,
+  refine ⟨λ i, is_empty_elim, λ i, _, is_empty_elim⟩,
+  rw ordinal.to_pgame_move_left',
+  exact IH _ (ordinal.to_left_moves_to_pgame_symm_lt i)
+end
```

Comment:
I'd argue this belong in `ordinal.lean`, especially since we have the TODO comment:

> - Extend this map to `game` and `surreal`.

there.

Can you add the `ordinal.to_surreal` map in this PR too? It looks like it's just a trivial application of `surreal.mk` with `numeric_to_pgame` lemma.

Reply:
Having this in `ordinal.lean` would mess up the dependencies. In my opinion, that file should be a basic file within the `game` folder, while the `surreal` folder should have access to most if not all of the `game` folder.

I do still think we should have more API for ordinal games and surreals, but I'm not sure how to go about doing this in organizational terms.

Reply:
Just wanted to check you'd explicitly ruled out putting it in `ordinal.lean`.

The request for `ordinal.to_surreal` still stands.